src/quo/core.cljs

```diff
@@ -402,7 +402,9 @@
 (def channel-name quo.components.text-combinations.channel-name.view/view)
 (def page-top quo.components.text-combinations.page-top.view/view)
 (def standard-title quo.components.text-combinations.standard-title.view/view)
-(def text-combinations quo.components.text-combinations.view/view)
+(def ^{:deprecated "quo.components.text-combinations.page-top.view should be used instead"}
+     text-combinations
+  quo.components.text-combinations.view/view)
```

Comment:
Deprecates the text-combinations component. It is not listed in designs and `page-top` should be used instead

Reply:
@FFFra did we have some approach for deprecating before? or is this the right one?